hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4352,6 +4352,17 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.client.pipeline.recovery.max-retries</name>
+  <value>5</value>
+  <description>
+    If we had to recover the pipeline more than the value
```

Comment:
Suggest to add a description for this configuration property. Something like "if the DFS client encounters errors in write pipeline, retry up to the number defined by this property before giving up"

Reply:
Good suggestion, i have changed to it.
Thanks @jojochuang

Reply:
Good suggestion, i have changed to it.
Thanks @jojochuang